core/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -357,4 +357,16 @@ NullValuePropertyMappingStrategy nullValuePropertyMappingStrategy() default
      * @since 1.4
      */
     Class<? extends Exception> unexpectedValueMappingException() default IllegalArgumentException.class;
+
+    /**
+     * Flag indicating whether the creation of a time stamp in the {@code @Generated} annotation should be suppressed.
+     * i.e. not be created.
+     *
+     * The method overrides the flag set in a central configuration set by {@link #config()}
+     *
+     * @return whether the creation of a timestamp should be suppressed
+     *
+     * @since 1.5
+     */
+    boolean suppressGeneratorTimestamp() default false;
```

Comment:
`suppressGeneratorTimestamp()` .. I guess `suppressGenerationTimestamp()` is better. It's the time-stamp of 'the act of' generation, right?

Reply:
I used the same property as the compiler option. If we are going for something else then perhaps `suppressTimestampGeneration` is better, at least it reads better to me than `suppressGenerationTimestamp`.

Reply:
ok.. leave as is..